crates/orchestrator/src/cli/cron/event_bridge.rs

```diff
@@ -0,0 +1,22 @@
+use clap::Args;
+
+/// Parameters used to config AWS SNS.
+#[derive(Debug, Clone, Args)]
+#[group()]
+pub struct AWSEventBridgeCliArgs {
```

Comment:
is this only used under the setup command?

Reply:
yes moved in the run/setup system